entity/entity.py

```diff
@@ -16,3 +33,19 @@ def render(self, screen, camera: Camera):
             )
         )
         screen.blit(self.graphic, rect)
+
+    def _position(self, dt):
+        pos_x += self.velocity_x * dt + .5 * self.accel_x * dt**2
+        pos_y = self.velocity_y * dt + .5 * self.accel_y * dt**2
+        self.position = (pos_x ,pos_y )
+
+    def _accelerate(self, dt):
+        if self.accel == True:
+            self.velocity_x += self.accel_x
+            self.velocity_y += self.accel_y
+
+    def _rotate(self, rotate_rate, dt):
+        if self.rotate_clockwise == True:
+            self.angle = (self.angle + rotate_rate * dt ) % 360
+        if self.rotate_counterclockwise == True:
+            self.angle = (self.angle - rotate_rate * dt ) % 360
```

Comment:
Same here. this should change the angle directly. No need for the boolean switches